sale_by_packaging/models/product_packaging.py

```diff
@@ -18,3 +17,10 @@ class ProductPackaging(models.Model):
         "When the user will put 3 as quantity, the system can force the "
         "quantity to the superior unit (5 for this example).",
     )
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
wouldn't be better a computed field w/ readonly=False? :thinking:

Reply:
also, any test?

Reply:
> wouldn't be better a computed field w/ readonly=False? :thinking:

It was my first implementation.
But during the discussion it was decided that a change on the product.packaging.type should not cascade to the product.packaging. As the product_packaging_type_id is a required field it is better done like this IMO

Reply:
hmm but then do not depend on it, no?

Reply:
I don't get what you mean, do not depend on what ?

Reply:
Implement it with an onchange instead of create

Reply:
So finally I implemented it with a compute readonly=False and store=True
Seems to work as expected.